system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -2533,6 +2533,7 @@ class CLdapClient : implements ILdapClient, public CInterface
             userTree->addProp(getUserFieldNames(UFFullName), fullName);
```

Comment:
legacy, but worth fixing the indentation issue here (fixing it won't confuse reviewer)